cctv/process_images.py

```diff
@@ -67,16 +65,18 @@ def create_camera(record, fallback_img):
     Raises:
         ValueError: raised by <Camera > if not ip, id, or model
     """
-    ip = record.get(IP_FIELD)
+    ip = record.get(IP_FIELD).strip()
```

Comment:
noticed some IPs in knack had trailing/leading spaces so this will clean these up